### DISCUSSION TITLE
CNC Machined Components?

### DISCUSSION THREAD

**Discussion by tshumay** [FreddieHong19/Open5x#39] [Ideas]
Hi Freddy - I just read about your project in Make. It's fascinating to me to imagine the potential for this type of printing. 

I have an in-house CNC machine shop here as part of my prototype shop. I was curious if you think it would be beneficial to create a machined aluminum version of your gantry and associated components for rigidity/durability? I'd be very interested in working on that if it made sense.

**Comment by FreddieHong19** [comment]
Yes! I am sure it is beneficial to create the part in CNC. It will be rigid and durable. I think it make sense to use aluminium if it's an available option.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Trent Shumay ***@***.***>
Sent: Monday, January 2, 2023 3:20:07 PM
To: FreddieHong19/Open5x ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [FreddieHong19/Open5x] CNC Machined Components? (Discussion #39)


This email from ***@***.*** originates from outside Imperial. Do not click on links and attachments unless you recognise the sender. If you trust the sender, add them to your safe senders list<https://spam.ic.ac.uk/SpamConsole/Senders.aspx> to disable email stamping for this address.



Hi Freddy - I just read about your project in Make. It's fascinating to me to imagine the potential for this type of printing.

I have an in-house CNC machine shop here as part of my prototype shop. I was curious if you think it would be beneficial to create a machined aluminum version of your gantry and associated components for rigidity/durability? I'd be very interested in working on that if it made sense.
